docker-compose.yaml

```diff
@@ -1,26 +1,58 @@
-version: '3.9.1'
+version: "3.9.1"
 
-# You must manually run `ankerctl config import` atleast once
-# docker run \
-# -v ankerctl_vol:/root/.config/ankerctl \
-# -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-# ankerctl config import /tmp/login.json
 services:
-  ankerctl:
-    image: ankerctl/ankerctl:latest
-    container_name: ankerctl
-    restart: unless-stopped
-    build: .
-    environment:
-      - FLASK_PORT=4470
-      - FLASK_HOST=0.0.0.0
-    volumes:
-      - ankerctl_vol:/root/.config/ankerctl
-    ports:
-      - 127.0.0.1:4470:4470
-    entrypoint: "/app/ankerctl.py"
-    command: ["webserver", "run"]
+    ankerctl_seed_mac:
+        image: busybox
+        container_name: ankerctl_seed_mac
+        restart: "no"
+        volumes:
+            - ankerctl_vol:/root/.config/ankerctl
+            - "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json"
```

Comment:
This will fail on mac because of the space, you need to properly escape the space between `Application` and `Support`.